week4tasks/src/week4/palindrom/Main.java

```diff
@@ -0,0 +1,40 @@
+package week4.palindrom;
+
+import java.util.Scanner;
+
+class palindrom {
+    public  String palindromstring(String str) {
```

Comment:
make this method return boolean not String as it is not good practice to return null